certified-connectors/Spinpanel/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2047 @@
+                  "type": "array",
+                  "items": {
+                    "description": "On premise provisioning errors of the group",
+                    "title": "On premise provisioning errors",
+                    "type": "string"
+                  },
+                  "description": "onPremisesProvisioningErrors"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create Azure AD group",
+        "operationId": "PostGraphGroup",
+        "parameters": [
+          {
+            "name": "organizationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Spinpanel organization id.",
+            "x-ms-summary": "Spinpanel organization id.",
+            "x-ms-url-encoding" : "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "description": {
+                  "type": "string",
+                  "description": "Description of the group.",
+                  "title": "Description"
+                },
+                "displayName": {
+                  "type": "string",
+                  "description": "Display name of the group.",
+                  "title": "Display name"
+                },
+                "groupTypes": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "Group types for the group.",
+                    "title": "Group types"
+                  },
+                  "description": "groupTypes"
+                },
+                "mailEnabled": {
+                  "type": "boolean",
+                  "description": "Indicates if the group is mail enabled.",
+                  "title": "Mail enabled",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "mailNickname": {
+                  "type": "string",
+                  "description": "Mail nickname of the group.",
+                  "title": "Mail nickname"
+                },
+                "securityEnabled": {
+                  "type": "boolean",
+                  "description": "Indicates if the group is security enabled.",
+                  "title": "Security enabled",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                }
+              }
+            }
+          }
+        ],
+        "description": "Create azure AD group."
+      }
+    },
+    "/directory/v1/organizations/{organizationId}/usergroups/{userGroupId}": {
+      "delete": {
+        "responses": {
+          "204": {
+            "description": "Spinpanel usergroup removed",
+            "schema": {}
```

Comment:
@VincentMejan Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Removed empty response schema where no actual content is returned by our APIs